packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -231,6 +231,12 @@ function onWindowClick(e: Event): void {
   {/if}
 
   <select use:buildOptions class="hidden" name={name} bind:value={value}>
-    {@render children?.()}
+    {#if !children || children.length === 0}
+      {#each options as option}
+        <option value={option.value}></option>
+      {/each}
```

Comment:
I could also do only `<option value={value}></option>` instead of the entire `each` block, with `{value}` being the dropdown's value (tried and it worked for the Kind creation forms), but I'm not sure what implications it might have with only one value that is dynamic and maybe the `bind:value={value}` part